README.md

```diff
@@ -3,6 +3,15 @@
 This project provides tooling and automation to integrate upstream open source
 projects into Fedora operating system.
 
+## Elevator pitch
+
+Packit  is a CLI tool that helps developers auto-package upstream projects
+ into Fedora operating system.
+With just one configuration file in your upstream repository,
+packit will automatically generate RPM packages for Fedora
+and update them as you update your source upstream.
```

Comment:
Lines 8 and 9 are pretty much a duplicate of lines 3 and 4.

I would replace lines 10-12 with the fact that you can use packit to continously build your upstream project in Fedora.